packages/nuxt/src/app/plugins/payload.client.ts

```diff
@@ -26,5 +18,15 @@ export default defineNuxtPlugin({
       if (!payload) { return }
       Object.assign(nuxtApp.static.data, payload.data)
     })
+
+    onNuxtReady(() => {
+      // Load payload into cache
+      nuxtApp.hooks.hook('link:prefetch', async (url) => {
+        if (!parseURL(url).protocol) {
+          await loadPayload(url)
+        }
+      })
+      setTimeout(getAppManifest, 1000)
```

Comment:
Would be nice if we also checked the connection speed as well. Performance "hopefully" is okay because hopefully, mobiles load the critical path in less than a second. But i am more worried about paid internet connections that waste bandwidth if they don't need the manifest.

Reply:
how do you think we should handle mobile or slow internet? increase the timeout?

Reply:
longer timeout seems a fine or also maybe conditionally disable here to lazily fetch it on CSR navigations? I see we await on it in middleware